src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,35 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+@WebServlet(urlPatterns = "/drivers/create")
```

Comment:
```suggestion
@WebServlet(urlPatterns = "/drivers/add")
```

Read common mistakes.